deploy/scripts/src/mobile/utils/functions-mobile-journey.ts

```diff
@@ -69,15 +89,22 @@ function isHeaderLocationCorrect (res: Response, content: string): boolean {
 }
 
 function postToVerifyURL (): Response {
-  return http.post(getUrl('start', env.testClientExecuteUrl),
+  return http.post(
+    getUrl('start', env.testClientExecuteUrl),
     JSON.stringify({ target: env.backEndUrl, frontendUri: env.frontEndUrl }),
-    { tags: { name: 'Post request to Verify URL' }, headers: { 'Content-Type': 'application/json' } }
+    {
+      tags: { name: 'Post request to Verify URL' },
+      headers: { 'Content-Type': 'application/json' }
+    }
   )
 }
 
 function parseVerifyUrl (response: Response): string {
   const responseBody = response.body?.toString()
-  const verifyUrl = typeof responseBody === 'string' ? JSON.parse(responseBody).WebLocation : null
+  const verifyUrl =
+    typeof responseBody === 'string'
```

Comment:
Just a thought would it be clearer if we do something like this: `if (responseBody){JSON.parse(responseBody).WebLocation}`

Reply:
Updated to use response.json here